README.md

```diff
@@ -503,3 +504,18 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
+4. From the master branch, run `git tag -a -m 'v1.8.10' 'v1.8.10'` replacing v1.8.10 with the new version found in step 3. Remember to keep the `v` prefix. So if the version in step 3 is `1.8.12` you would use `v1.8.12` in the git tag command in this step.
+5. Push the tag up to Github `git push upstream v1.8.10`.
```

Comment:
The new tag can actually be created during drafting the release

Reply:
I fixed the directions.